receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/label.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecscontainermetrics
+
+import (
+	"strings"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+)
+
+func containerLabelKeysAndValues(cm ContainerMetadata) ([]*metricspb.LabelKey, []*metricspb.LabelValue) {
+	labelKeys := make([]*metricspb.LabelKey, 0, 3)
+	labelValues := make([]*metricspb.LabelValue, 0, 3)
+
+	labelKeys = append(labelKeys, &metricspb.LabelKey{Key: "ecs.container-name"})
```

Comment:
We have semantic conventions for container, we should use them instead of custom `ecs.` prefix

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/container.md

Reply:
We had an internal design meeting and we agreed to follow this convention. I can update it following your suggestion. For container labels, I will prefix with `container.` and for task labels with `task.`.

Reply:
Since task is an ECS-specific concept, `ecs.cluster` for example is fine. We should follow the semantic conventions for common concepts though, so these will be `container.*` but task can stick to `ecs.*` I think

Reply:
I agree. One more thing, our customer can get similar metrics from different sources. For example, customer can get `CpuUtilized` and `MemoryUtilized` from TACS as well as using AOC from ECS Task Metadata Endpoint. To differentiate our Task Metadata Endpoint metrics from other metrics, we planned to prefix them with `ecs.container`. I am OK with either option.